# BLSSpeller_validation
The regulation of gene expression is hidden in non-coding regions, containing clusters of DNA binding sites recognized by transcription factors (TFs). Therefore, pinpointing the location of the functional TF binding sites, which are short and degenerate, can broaden our knowledge of the general picture of the complex regulatory network, and can facilitate application to induce phenotypic variation via genome editing programs. We incorporated the relationship between homologs among species in motif identification using the phylogenetic footprinting approach. BLSSpeller was redesigned to accommodate more species in the comparative analysis to increase the resolution of the phylogenetic shadowing. Like the results from any other computation approach, the conserved motif predicted by BLSSpeller requires extensive evaluation. The redesigned BLSSpeller was applied to discover novel cis-elements in Zea mays. This collection of R, python, and julia codes shows how the predicted motifs can be validated using genomics and functional data.
